src/stackdriver-nozzle/config/config.go

```diff
@@ -0,0 +1,42 @@
+package config
```

Comment:
I get where you're coming from. We did this to support the scripts, but I think I'm okay having the scripts just work off environment vars anyway.